packages/js/email-editor/src/components/personalization-tags/category-section.tsx

```diff
@@ -52,7 +52,7 @@ const CategorySection = ( {
 						<div className="woocommerce-personalization-tags-modal-category-group">
 							{ items.map( ( item ) => {
 								// TODO: Improve logic for detecting URL tags.
```

Comment:
Do we still need this here? If it could be deleted, it would be better, but if not, I think we need more details.

Reply:
I understand your reasoning, but I don't find the comment very helpful. It doesn't explain it. Based on my previous experience, TODO comments often remain in the codebase for years without anyone taking action on them. Anyway, I am OK with keeping it here, but I would try to explain better what exactly "Improve logic" means. Because without any context, it is hard to understand.

Reply:
Updated the comment to be more descriptive and removed that "TODO" part as it is indeed not very useful.